frontend/packages/dev-console/integration-tests/testData/git-import/repos.ts

```diff
@@ -31,5 +181,12 @@ export function getResponseMocks(repo: GithubRepo) {
   } catch (err) {
     // nothing, the file does not exist
   }
-  return { repoResponse: repoJson, contentsResponse: contentsJson, packageResponse: packageJson };
+  return {
+    repoResponse: repoJson,
+    contentsResponse: contentsJson,
+    packageResponse: packageJson,
+    ...(repo.url === 'https://github.com/nodeshift-starters/devfile-sample' && {
+      devFileResources,
+    }),
```

Comment:
Then you can avoid the special case here:

```suggestion
    devFileResources: repo.devFileResources,
```